src/clj/puppetlabs/trapperkeeper/services/protocols/filesystem_watch_service.clj

```diff
@@ -6,9 +6,13 @@
   "Schema for an event on a file watched by this service."
   (schema/if #(= (:type %) :unknown)
     {:type (schema/eq :unknown)
-     :path (schema/pred nil?)}
+     :count schema/Int
+     :watched-path File}
     {:type (schema/enum :create :modify :delete)
-     :path File}))
+     :count schema/Int
+     :watched-path File
+     :changed-path File
+     :full-path File}))
```

Comment:
We had a comment thread going which I think got lost in [an outdated diff](https://github.com/puppetlabs/trapperkeeper-filesystem-watcher/pull/8#discussion_r73890035).

To summarize:

It's pretty weird to return two separate `File` objects for a single path on the filesystem.  That makes me wonder if we could just have a single `:changed-path` which is absolute.  Is there any specific need for the relative path to show up in these `Event` objects?